docs: minor simplification of anti-patterns page (#13293)
elixir-lang/elixir: lib/elixir/pages/anti-patterns/code-anti-patterns.md

end
```

> While the example above is not an anti-pattern because it is so small, it is an example of mixed extraction and pattern matching. A situation where `drive/1` was more complex, having many more clauses, arguments, and extractions, would make it hard to know at a glance which variables are used for pattern/guards and which ones are not.

Reviewer: I think the intended word was `constitute`? (I'm not sure)

```suggestion
While the example above is small and does not constitute an anti-pattern, it is an example of mixed extraction and pattern matching. A situation where `drive/1` was more complex, having many more clauses, arguments, and extractions, would make it hard to know at a glance which variables are used for pattern/guards and which ones are not.
```
